src/components/qna-components/answer-utility/qna-answer-utility.component.jsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import propTypes from 'prop-types';
+import Moment from 'moment';
+import './qna-answer-utility.styles.scss';
+
+class AnswerUtility extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      answerHelpfulness: 0,
+      answerHelpfulClicked: false,
+    };
+  }
+
+  componentDidMount() {
+    const { answerHelpfulness } = this.props;
+    this.setState({ answerHelpfulness });
+  }
+
+  updateHelpful = (e) => {
+    const { answerId } = this.props;
+    e.preventDefault();
+    this.setState((previousState) => ({
+      answerHelpfulness: previousState.answerHelpfulness + 1,
+    }),
+    () => {
+      const { answerHelpfulClicked } = this.state;
+      this.setState({ answerHelpfulClicked: !answerHelpfulClicked }, () => {
+        fetch(`http://3.134.102.30/qa/answer/${answerId}/helpful`,
+          { method: 'PUT' });
+      });
```

Comment:
if this errs, do you want to change the value of answer helpfulness back?

Reply:
@danielmjordan that's a great point, even though when the user refreshes it would show the correct value (on get fetch), it would be accurate if it erred. I may exclude from MVP, but noting it as a item to come back to. appreciate you catching that!